src/azure/cli/commands/parameters.py

```diff
@@ -73,6 +73,19 @@ def completer(prefix, action, parsed_args, **kwargs): # pylint: disable=unused-a
 def get_enum_choices(enum_type):
     return [x.value for x in enum_type]
 
+class IgnoreAction(argparse.Action): # pylint: disable=too-few-public-methods
+    def __call__(self, parser, namespace, values, option_string=None):
+        raise argparse.ArgumentError(None, 'unrecognized argument: {} {}'.format(
+            option_string, values or ''))
+
+# CONSTANTS
+
+IGNORE_TYPE = CliArgumentType(
```

Comment:
I used all caps so it would be similar to argparse.SUPRESS. It signifies that it does something a bit different.

---
In reply to: [76151354](https://github.com/Azure/azure-cli/pull/744#discussion_r76151354) [](ancestors = 76151354)